backend/controller/observability/pubsub.go

```diff
@@ -12,15 +12,17 @@ import (
 	"go.opentelemetry.io/otel/metric/noop"
 
 	"github.com/TBD54566975/ftl/backend/schema"
+	"github.com/TBD54566975/ftl/internal/model"
 	"github.com/TBD54566975/ftl/internal/observability"
 )
 
 // To learn more about how sinks and subscriptions work together, check out the
 // https://tbd54566975.github.io/ftl/docs/reference/pubsub/
 const (
 	pubsubMeterName              = "ftl.pubsub"
-	pubsubTopicNameAttr          = "ftl.pubsub.topic.name"
-	pubsubCallerVerbNameAttr     = "ftl.pubsub.publish.caller.verb.name"
+	pubsubTopicRefAttr           = "ftl.pubsub.topic.ref"
+	pubsubTopicModuleAttr        = "ftl.pubsub.topic.module.name"
```

Comment:
How is this different to .ref above?

Reply:
Oh hmm it looks like DD's "split graph" feature doesn't support any special string matching; you have to just pick an attribute and it slices all the graphs by that. So that would be an argument to keep module in there :/